Gems/AtomLyIntegration/AtomViewportDisplayIcons/Code/Source/AtomViewportDisplayIconsSystemComponent.cpp

```diff
@@ -354,20 +361,22 @@ namespace AZ::Render
     {
         // Once the shader is loaded, register it with the dynamic draw context
         Data::Asset<RPI::ShaderAsset> shaderAsset = asset;
-        AtomBridge::PerViewportDynamicDraw::Get()->RegisterDynamicDrawContext(m_drawContextName, [shaderAsset](RPI::Ptr<RPI::DynamicDrawContext> drawContext)
+        AtomBridge::PerViewportDynamicDraw::Get()->RegisterDynamicDrawContext(m_drawContextName, [shaderAsset](RPI::Ptr<RPI::DynamicDrawContext> dynamicDraw)
```

Comment:
nit: updating the variable name seems unnecessary, the new name is not vastly more correct, so this just seems like a personal preference change.

Reply:
The reason for this is in all other places the name is referred to as `dynamicDraw` (if you Ctrl-F in the file you can find other usages) and all conversations I had with the Atom team about it referred to it as this which is why I decided it made sense to change for consistency. I would rather keep the change as it makes finding usages easier when grepping for that string (which I do often when trying to understand a new area of code)